examples/cluster_scoped_custom_object.py

```diff
@@ -0,0 +1,147 @@
+            spec:
+              type: object
+              properties:
+                cronSpec:
+                  type: string
+                image:
+                  type: string
+                replicas:
+                  type: integer
+  scope: Cluster
+  names:
+    plural: crontabs
+    singular: crontab
+    kind: CronTab
+    shortNames:
+    - ct
+"""
+
+from pprint import pprint
+
+from kubernetes import client, config
+
+
+def main():
+    config.load_kube_config()
+
+    api = client.CustomObjectsApi()
+
+    # definition of custom resource
+    test_resource = {
+        "apiVersion": "stable.example.com/v1",
+        "kind": "CronTab",
+        "metadata": {"name": "test-crontab"},
+        "spec": {"cronSpec": "* * * * */5", "image": "my-awesome-cron-image"},
+    }
+
+    # patch to update the `spec.cronSpec` field
+    cronspec_patch = {
+        "spec": {"cronSpec": "* * * * */15", "image": "my-awesome-cron-image"}
+    }
+
+    # patch to add the `metadata.labels` field
+    metadata_label_patch = {
+        "metadata": {
+            "labels": {
+                "foo": "bar",
+            }
+        }
+    }
+
+    # create a cluster scoped resource
+    created_resource = api.create_cluster_custom_object(
+        group="stable.example.com",
+        version="v1",
+        plural="crontabs",
+        body=test_resource,
+    )
+
+    # get the cluster scoped resource
+    resource = api.get_cluster_custom_object(
+        group="stable.example.com",
+        version="v1",
+        name="test-crontab",
+        plural="crontabs",
+    )
+    print("%s\t\t%s" % ("NAME", "SCHEDULE"))
+    print(
+        "%s\t%s\n" %
+        (resource["metadata"]["name"],
+         resource["spec"]["cronSpec"]))
+
+    # patch the `spec.cronSpec` field of the custom resource
+    patched_resource = api.patch_cluster_custom_object(
+        group="stable.example.com",
+        version="v1",
+        plural="crontabs",
+        name="test-crontab",
+        body=cronspec_patch,
+    )
+    print("%s\t\t%s" % ("NAME", "PATCHED_SCHEDULE"))
+    print(
+        "%s\t%s\n" %
+        (patched_resource["metadata"]["name"],
+         patched_resource["spec"]["cronSpec"]))
+
+    # patch the `metadata.labels` field of the custom resource
+    patched_resource = api.patch_cluster_custom_object(
+        group="stable.example.com",
+        version="v1",
+        plural="crontabs",
+        name="test-crontab",
+        body=metadata_label_patch,
+    )
+    print("%s\t\t%s" % ("NAME", "PATCHED_LABELS"))
+    print(
+        "%s\t%s\n" %
+        (patched_resource["metadata"]["name"],
+         patched_resource["metadata"]["labels"]))
+
+    # delete the custom resource "test-crontab"
+    api.delete_cluster_custom_object(
+        group="stable.example.com",
+        version="v1",
+        name="test-crontab",
+        plural="crontabs",
+        body=client.V1DeleteOptions(),
+    )
+    print("Resource `test-crontab` deleted!")
```

Comment:
Could you add similar log to create and patch? Thanks

Reply:
Added log messages for above suggestions. Thank you.